topsort-api-v2.yml

```diff
@@ -111,20 +79,21 @@ paths:
                   items:
```

Comment:
Please specify the maximum number of simultaneous auctions we are going to support.